apps/admin/frontend/index.html

```diff
@@ -1,21 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Another election tool from VotingWorks" />
+    <link rel="stylesheet" href="/fonts/helvetica-neue.css" />
+    <link rel="stylesheet" href="/fonts/noto-emoji.css" />
+    <link rel="stylesheet" href="/fonts/edu-tas-beginner.css" />
+    <title>VotingWorks VxAdmin</title>
+  </head>
 
-<head>
-  <meta charset="utf-8" />
-  <link rel="icon" href="/favicon.ico" />
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="theme-color" content="#000000" />
-  <meta name="description" content="Another election tool from VotingWorks" />
-  <link rel="stylesheet" href="/fonts/helvetica-neue.css" />
-  <link rel="stylesheet" href="/fonts/noto-emoji.css" />
-  <link rel="stylesheet" href="/fonts/edu-tas-beginner.css" />
-  <title>VotingWorks VxAdmin</title>
-</head>
-
-<body>
-  <div id="root"></div>
-  <script type="module" src="./src/index.tsx"></script>
-</body>
-
-</html>
\ No newline at end of file
+  <body>
+    <div id="root" class="no-print"></div>
```

Comment:
Note to self: I think this may have been something I was trying but ultimately rolled back, so this class could be removed.

Reply:
removed